nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ConsumeEWS.java

```diff
@@ -107,6 +109,14 @@
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .sensitive(true)
             .build();
+    public static final PropertyDescriptor MAILBOX = new PropertyDescriptor.Builder()
+            .name("mailbox")
+            .displayName("Mailbox")
+            .description("Mailbox")
+            .required(true)
```

Comment:
If your going to make MAILBOX required, then you'll need to include a default value so that it doesn't break existing users.  Or make it not required, and adjust the code so that it works the way it used to if no MAILBOX is provided.

Reply:
made some changes to in below commits